src/task2.js

```diff
@@ -0,0 +1,31 @@
+import csv from 'csvtojson';
+import { createReadStream, createWriteStream } from 'fs';
+import { pipeline } from 'stream';
+
+const inputFilePath='./csv/nodejs-hw1-ex1.csv';
+const outputFilePath='./csv/nodejs-hw1-ex1.txt';
+
+
+const readable = createReadStream(inputFilePath);
+const writable = createWriteStream(outputFilePath, 'utf8');
+
+// Solution 1
+// readable
+//   .pipe(csv())
+//   .on('error', err => console.log(err))
+//   .pipe(writable)
+//   .on('error', err => console.log(err));
+
+// Solution 2
+pipeline(
+  createReadStream(inputFilePath),
```

Comment:
*Do not load all the content of the csvfile into RAM via stream (read/write file content **line by line**).*
You read all file. Write works well, because `csvToJson` emit line by line

![image](https://user-images.githubusercontent.com/17414540/112130164-3c412b00-8bd9-11eb-82f6-14c527bd5251.png)